lm_eval/tasks/super_glue/boolq/default.yaml

```diff
@@ -7,12 +7,7 @@ output_type: multiple_choice
 training_split: train
 validation_split: validation
 doc_to_text: "{{passage}}\nQuestion: {{question}}\nAnswer:"
```

Comment:
I might be mistaken, but the old version has a question mark after the question, i.e.
```yaml
doc_to_text: "{{passage}}\nQuestion: {{question}}?\nAnswer:"
```